pyproject.toml

```
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Contributors to the OpenEXR Project.
+
+[build-system]
+requires = ["scikit-build-core==0.8.0"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "OpenEXR"
+dynamic = ["version"]
+
+description="Python bindings for the OpenEXR image file format"
+readme = "src/wrappers/python/README.md"
+authors = [
+  { name="Contributors to the OpenEXR project", email="info@openexr.com" },
+]
+requires-python = ">=3.7"
+
+[project.urls]
+"Homepage" = "https://openexr.com"
+"Source" = "https://github.com/AcademySoftwareFoundation/OpenEXR"
+"Bug Tracker" = "https://github.com/AcademySoftwareFoundation/OpenEXR/issues"
+
+[project.optional-dependencies]
+test = ["pytest"]
+
+[tool.scikit-build]
+wheel.expand-macos-universal-tags = true
+wheel.packages = ["src/wrappers/python"]
+
```

```suggestion
sdist.exclude = [".github", "src/test", "src/examples", "website", "ASWF", "bazel", "share"]
```
This allows to reduce the size of the generated sdist from 18M to 1.6M. Laternatively, we could use `sdist.include` to only include what we need instead of excluding what we don't need.

As a side note, I would add `dist/*` to the `.gitignore` to avoid having the generated artifacts be included in the sdist by accident when building locally.